src/content/about/history.mdx

```diff
@@ -0,0 +1,27 @@
+import { IllustrationTextSection } from "../../components/sections/illustrationTextSection.js";
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper";
+import about1 from "../../images/about/Europa-center.png";
+import about2 from "../../images/about/streamline.png";
+
+<MaxWidthWrapper padding="1rem 2rem 1rem 2rem" backgroundColor="#FFFFFF" gutterBottom>
```

Comment:
the `padding` prop actually doesn't get passed into the `<MaxWidthWrapper/>` component, so it can be safely removed